Dockerfile.dev

```diff
@@ -3,10 +3,28 @@ FROM python:3.6-alpine
 RUN apk update && apk add python3-dev build-base git
 
 WORKDIR /app
+
+# Some dependencies require a very long compilation: protobuf, numpy,
+# grpcio. To avoid having to re-install these packages every time, we
+# pre-install them so that they are cached.
+#
+# However, we cannot just pre-install a few packages, because we don't
+# know exactly which version `pip` will pick for them. Instead, we
+# give pip all the package's dependencies, and let it resolve and
+# install them.
 COPY setup.py .
-COPY xain_fl xain_fl/
+RUN mkdir xain_fl && \
+    printf '__version__ = "0.2.0"\n__short_version__ = "0.2"' > xain_fl/__version__.py && \
```

Comment:
It shouldn't make much of a difference, but at the moment of this writing `v0.3.0` has already been released for `xain-fl`: https://github.com/xainag/xain-fl/pull/230

Reply:
These are dummy numbers anyway. I should perhaps make it clearer by making it just "0".

Reply:
Done. Changed it to `0`.